presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -398,6 +399,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         OFFLINE_DATA_DEBUG_MODE_ENABLED,
                         "allow reading from tables or partitions that are marked as offline or not readable",
                         false,
+                        true),
+                booleanProperty(
+                        ORC_ZSTD_JNI_DECOMPRESSION_ENABLED,
+                        "use JNI based zstd decompression for reading ORC files",
+                        false,
                         true));
```

Comment:
This session property is visible I assume?

Reply:
AS i understand, visible means "visible to the user". We don't really need it to be visible to the user based on this interpretation.

Reply:
The parameter says hidden, and so it's marked true. We don't want to surface it to the user.